custom_student_verification/locale/en/LC_MESSAGES/django.po

```diff
@@ -39,3 +54,75 @@ msgstr ""
 #: templates/custom_student_verification.html
 msgid "Submit"
 msgstr ""
+
```

Comment:
@alqahtani I've just added the remaining translations here, can you provide translations for the remaining strings?

Reply:
this is inside "en" folder! are you sure it's the correct one?

Reply:
this is inside "en" folder! are you sure it's the correct one?

Reply:
@alqahtani Ah, I've added the extra strings to the `en` file (retrieved automatically).
You can copy them to the `ar` folder and supply the missing translations there.